File: arm.64/langs/csharp/src/trip.cs
namespace Pat;

public static class Tripper
{
    public static List<Triplet> Trip()
    {
        return new List<Triplet>
        {
            new Triplet { Address = "/version", Payload = "{}", Options = "{\"once\":true}" },
            new Triplet { Address = "/storage", Payload = "{}", Options = "{\"once\":true}" },
            new Triplet { Address = "sql.help", Payload = "{}", Options = "{\"once\":true}" }
        };
    }
}

File: arm.64/langs/go/src/trip.go
package main

func trip() []Triplet {
	return []Triplet{
		{Address: "/version", Payload: "{}", Options: `{"once":true}`},
		{Address: "/storage", Payload: "{}", Options: `{"once":true}`},
		{Address: "sql.help", Payload: "{}", Options: `{"once":true}`},
	}
}
